src/pages/mentorship/feedback-section.tsx

```diff
@@ -0,0 +1,94 @@
+// path: /mentorship/faqs
+
+import { Typography, Button, Box, Grid } from '@mui/material';
+import React, { useState } from 'react';
+
+import { ColoredBox, FeedbackCard } from '@components';
+import { FeedbackCardProps } from 'components/FeedbackCard';
+
+interface FeedbackSectionProps {
+  title: string;
+  feedbacks: FeedbackCardProps[];
+}
+
+export const FeedbackSection: React.FC<FeedbackSectionProps> = ({
```

Comment:
I am not sure how you doing, but I am missing the init test regarding this new component.